test/customer_repository_test.rb

```diff
@@ -1,18 +1,17 @@
-require 'minitest/autorun'
-require 'minitest/pride'
+require_relative 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.